### DISCUSSION TITLE
closed (no one is publishing to path 'art88')

### DISCUSSION THREAD

**Discussion by eqphp** [bluenviron/mediamtx#1083] [Questions]
![image](https://user-images.githubusercontent.com/5820284/184075580-11bed298-d348-4335-8f4d-18848f53d54c.png)

**Comment by aler9** [comment]
Hello, there's probably a firewall/NAT between the server and that publisher that blocks UDP packets.

Switch to TCP by reading [here](https://github.com/aler9/rtsp-simple-server#tcp-transport).

[a comment by an automated account is not shown]
